config/vufind/BrowZine.ini

```
@@ -68,6 +75,26 @@ fullTextFile = "PDF Full Text|browzine-pdf|https://assets.thirdiron.com/images/i
 retractionNoticeUrl = "View Retraction Notice|browzine-retraction"
 expressionOfConcernNoticeUrl = "View Expression of Concern Notice|browzine-concern"
 
+; The bestIntegratorLink service provides "the best link available" according to
+; Third Iron's algorithm. See
```

Worth adding something like this?

```suggestion
; Third Iron's algorithm. When activating this, you should comment out other options above. See
```

Not sure this exact wording is appropriate -- but I imagine we either want to display all relevant links, or else we want to display the "best" link, but uncommenting everything results in undesirable behavior.

(I also figured that adding some text here makes the long URL below look a bit better-balanced).

---

This is a good bit of advice, but the truth is a bit more subtle, something like 

> When activating this, you should comment out other options above that deal with this specific article, i.e. all except `browzineWebLink` which is a link to the full issue.  In practice you probably don't want that link either, because who care about reading full journal issues anymore, but maybe that's you!

I added a version of that with less snark.

---

I like your approach very much (even if the snark is more fun ;-) ) -- thanks!